src/lib/helpers.js

```diff
@@ -13,19 +15,10 @@ export const getAccountLogin = account => {
   }
 }
 
-export const getRandomKeyString = () =>
-  Math.random()
-    .toString(36)
-    .substring(2, 15) +
-  Math.random()
-    .toString(36)
-    .substring(2, 15) +
-  Math.random()
-    .toString(36)
-    .substring(2, 15)
+export const getRandomKeyString = () => uuid()
```

Comment:
`export { uuid as getRandomKeyString } from 'uuid/v4'` ?

Reply:
I will replace this getRandomKeyString usage, it will be cleaner